debian/control

```diff
@@ -12,5 +12,7 @@ Architecture: all
 Section: admin
 Priority: standard
 Depends: ${misc:Depends},
+         rsetup,
+         systemd,
```

Comment:
```suggestion
```
If we ever support multiple init systems, it should be `rsetup` to handle the difference between them.